### PR TITLE
Add docs for py-event*

### PR DESCRIPTION
**Pull request by marimeireles**
## Description

Please describe the changes in your pull request in few words here.

### Changes

List the changes done to fix a bug or introducing a new feature.

## Checklist

-   [x] All tests pass locally
-   [x] I have updated `docs/changelog.md`
-   [ ] I have created documentation for this(if applicable)

### CODE REVIEW

**Comment by marimeireles** [comment]
@dj-fiorex took the liberty of rebasing and attending to Jeff and Antonio's PR here. Hope it's not an issue. Thank you for the work here!
@antocuni can you please check if this is fine? I've rewrote the sentence you commented on and remove the duplication.
Cheers folks.

**Comment by antocuni** [comment]
> @dj-fiorex took the liberty of rebasing and attending to Jeff and Antonio's PR here. Hope it's not an issue. Thank you for the work here!

which PR?

Moreover, this PR should have a more descriptive title, because as it stands I have no idea of what it is supposed to do.

**Comment by marimeireles** [comment]
@antocuni sorry should have given more context.
The related issue is tagged above and the original PR is https://github.com/pyscript/pyscript/pull/852 you've reviewed months ago.
It's still relevant, imo, as long as I don't merge my PR on the new event handling stuff.

File: docs/guides/event-handlers.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+</button>
+<button id="withParam" py-click="say_hello_with_param('World')">
+ No Event - With Params py-click
+</button>
+```
+
+```python
+<py-script>
+    def say_hello_no_param():
+        print("Hello!")
```
Comment (fpliger) [inline docs/guides/event-handlers.md:21]: very minor but would be goo to add a new line between the 2 functions definition

**Review by fpliger — APPROVED** [review]
Thanks @marimeireles @dj-fiorex , this is overall a good PR. It introduces a few sections that are Pyodide specific and we'll have to change/reshape once we better support MicroPython and add the new Workers + FFI interface.

👍

File: docs/guides/event-handlers.md (hunk 1 above)
Comment (marimeireles) [reply docs/guides/event-handlers.md:21]: thanks!
Will fix so we can merge it :)

**Pull request merged**